frontend/packages/topology/src/components/DefaultCreateConnector.tsx

```diff
@@ -1,40 +1,66 @@
 import * as React from 'react';
-import Point from '../geom/Point';
+import * as classNames from 'classnames';
+import { AddCircleOIcon } from '@patternfly/react-icons';
+import { Tooltip } from '@patternfly/react-core';
 import ConnectorArrow from './ConnectorArrow';
+import { CreateConnectorRenderer } from './CreateConnectorWidget';
```

Comment:
`CreateConnectorWidget` imports `DefaultCreateConnector`
and `DefaultCreateConnector` imports  `CreateConnectorWidget`
Probably worth combining the two together.